package.json

```diff
@@ -29,6 +29,7 @@
     "rollup-plugin-node-resolve": "^2.0.0",
     "rollup-plugin-replace": "^1.1.1",
     "sass-lint": "^1.10.2",
+    "shx": "^0.2.1",
```

Comment:
I'm running on Windows, so I can verify it works on at least one configuration.  😉